dataStructures/structures/queue.js

```diff
@@ -0,0 +1,45 @@
+export default class Queue {
+  constructor() {
+    this.content = []
+    this.length = 0
+  }
+
+  enqueue(element) {
+    if(this.content === null) this.content = []
+    this.content.push(element)
+    this.length += 1
+  }
+
+  dequeue() {
+    if(this.length === 0) {
+      this.content = null
```

Comment:
Changing content from `[]` to `null` confuses me. I'd rather we return null and leave content as a `[]`